src/transformers/models/t5/modeling_flax_t5.py

```diff
@@ -966,6 +971,7 @@ def __call__(
         train: bool = False,
         params: dict = None,
         dropout_rng: PRNGKey = None,
+        input_embeds: Optional[jnp.ndarray] = None
```

Comment:
Same again regarding the name

```suggestion
        inputs_embeds: Optional[jnp.ndarray] = None
```